packages/document/module-doc/docs/en/api/config/build-config.md

```diff
@@ -284,6 +284,16 @@ Generate only dts files, not js files
 - Type: `boolean`
 - Default: `false`
 
+### respectExternal
+
+When set to `false`, three-party package types will be excluded from bundle, when set to `true`, it will determine whether three-party types need to be bundled based on[externals](#externals).
```

Comment:
```suggestion
When set to `false`, the type of third-party packages will be excluded from the bundle, when set to `true`, it will determine whether third party types need to be bundled based on [externals](#externals).
```